services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -106,7 +107,7 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
       }
 
       "return none if project not found" taggedAs Service in {
-        val projectId = ProjectId("projectId")
+        val projectId = ProjectId.random
```

Comment:
IMO it is a bit inconsistent that you are removing `id = ProjectConfigurationId.randomId` in `ProjectConfigurationServiceTestImpl` while adding it here. We should stick to either of this approach, but not change them back and forth (`randomId` looks better IMO)